source/user-guide/export/index.rst

```diff
@@ -43,4 +45,73 @@ It is possible to override the brief summary of metadata elements by creating a
    </metadata>
  </xsl:template>
 
-This template, when added to ``GEONETWORK_DATA_DIR/config/schema_plugins/iso19139/present/metadata-iso19139.xsl``, will replace the brief summary (produced by the brief template) with just one element, gmd:title.
+This template, when added to ``GEONETWORK_DATA_DIR/config/schema_plugins/iso19139/present/metadata.xsl``, will replace the brief summary (produced by the brief template) with just one element, gmd:title.
+
+In the case of iso19115-3.2018 there are the two files to be modified:
+
+1. geonetwork/WEB-INF/data/config/schema_plugins/iso19115-3.2018/layout/tpl-csv.xsl
+2. geonetwork/xslt/services/csv/csv-search.xsl
+
+For example, to export the following elements:
+
+``"uuid","title","cloud_coverage_percentage","category","date-creation"``
+
+In the first file, you have to add a similar line in between the abstract and category elements:
+::
+     <myfield>myfieldvalue</myfield>
+
+For example:
+
+::
+
+    <title>
+       <xsl:copy-of select="mdb:identificationInfo/*/mri:citation/*/cit:title"/>
+    </title>
+    <cloud_coverage_percentage>
+       <xsl:copy-of select="mdb:contentInfo/mrc:MD_ImageDescription/mrc:cloudCoverPercentage/gco:Real"/>
+    </cloud_coverage_percentage>
+    <category>
+      <xsl:value-of select="mdb:metadataScope/*/mdb:resourceScope/*/@codeListValue"/>
+    </category>
+
+About cloud coverage percentage, in particular, please remember to add the namespace if missing:   
+
+``xmlns:mrc="http://standards.iso.org/iso/19115/-3/mrc/2.0"``
+
+If not interested in other information, please comment all other lines except 
```

Comment:
Indeed geonet:info is used to sort results by schema https://github.com/geonetwork/core-geonetwork/blob/master/web/src/main/webapp/xslt/services/csv/csv-search.xsl#L120. One schema may have different columns from the others.So it is really important to not to remove that.

Reply:
Ok this is a better explanation why users should not comment that line